packages/-ember-data/tests/integration/adapter/rest-adapter-test.js

```diff
@@ -2576,10 +2576,6 @@ module('integration/adapter/rest_adapter - REST Adapter', function(hooks) {
         hash.error(jqXHR, jqXHR.responseText, errorThrown);
       };
 
-      adapter.handleResponse = function(status, headers, payload) {
```

Comment:
why did we need to delete this? Are we now calling this method *more* often than we used to? (note how this is in a `hasJQuery` branch)

Reply:
Yes. We are now ensuring we are calling it always, whether fetch or jquery. Let me know if this is not correct.

Reply:
Update: I now no longer call it in this situation (a non JSON string; status 200)